index.html

```diff
@@ -22,7 +22,7 @@
         border: solid 1px silver;
         background-color: whitesmoke;
         padding: 15px;
-      }
+      } 
       #title,
       #description {
         text-align: center;

```

Comment:
It's preferred to add create CSS rules using tags not Id of elements. for exampale using `h1` is more desirable than `#title`.